governance/polls/Ratification Poll - Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ratification Poll for Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022
+summary: This subproposal is requesting a 12 month Budget of 7,590,000 DAI for the Protocol Engineering Core Unit starting from May 2022.
```

Comment:
If we're gonna include the start date here, we should also include the end date, I think